spec/features/user_posts_tattoo_spec.rb

```diff
@@ -1,20 +1,66 @@
 require 'rails_helper'
 
-feature "User posts a tattoo to be reviewed", %(
-  As a user
-  I want to submit a tattoo for review
-  So that I could get feedback from others
+feature "User submits a tattoo of his own", %(
```

Comment:
Unless the spec is gender-specific, try to stick to a more neutral 'they/their'.